### DISCUSSION TITLE
Token names must be unique

### DISCUSSION THREAD

**Discussion by chocofrappe** [six7/figma-tokens#1351] [Q&A]
I have tried to follow the steps from How to get started on youtube. But I got the message "Token names must be unique" and cannot create this token I tried to name the sizing token as "small" which is the same as I already set for border radios in the previous steps. Does anyone know how to handle this case? Or did I choose to set it as a completely different name between token types? What if I want to set the sizing, border-radius as small? How can I do that?

![Screen Shot 2565-10-17 at 18 29 08](https://user-images.githubusercontent.com/108794411/196166012-9cf531cc-7c68-492a-92ce-f995575731a9.png)

**Comment by six7** [comment]
Hey @chocofrappe - token names are not tied to a specific group, so you best name them e.g. `sizing.small`, `spacing.small` etc
